### DISCUSSION TITLE
Code of Conduct link in Issue template goes to example.com

### DISCUSSION THREAD

**Discussion by rnikoopour** [goreleaser/goreleaser#2654] [Ideas, Issue Triage, and General discussions]
Hey All,

When opening a new issue I was going through the check list.  When I selected the link to read the code of conduct prior to agreeing it went to `example.com`.

Can this be updated with a proper code of conduct?

**Comment by caarlos0** [comment]
oh, good catch!

bad copy and paste from me, fixed on https://github.com/goreleaser/goreleaser/commit/9af17a49adde674d73c429daec7775d6156495a0
